pypuck/pypuck.py

```diff
@@ -101,11 +101,98 @@ def attendance(regular=True, playoffs=True, season=None):
     Examples
     --------
     >>> from pypuck import pypuck
-    >>> pypuck.attendance(regular=True, playoffs=True, season='2019-02')
+    >>> pypuck.attendance(regular=True, playoffs=True, start_season=2000, end_season=2019)
         
     ...
     """
-    pass
+    
+    # Specify the URL
+    url= 'https://records.nhl.com/site/api/attendance'
+
+    # Make the API request
+    page = requests.get(url)
+
+    df = pd.DataFrame(page.json()['data']).sort_values(by=['seasonId'])
+
+    # Check the response code is valid - i.e. the API didn't fail
+    helpers.check_response_code(page.status_code)
```

Comment:
@chuusan Thanks for adding in the helper!! One minor thing. We want the helper to be called before getting the df, so that the function gracefully errors if there is a bad response code:

```suggestion
    # Check the response code is valid - i.e. the API didn't fail
    helpers.check_response_code(page.status_code)
    
    df = pd.DataFrame(page.json()['data']).sort_values(by=['seasonId'])
```